HW1_data_explore.md

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Hi, great job with you work!! I don't think your files are actually in the hw1 folder though you may want to fix that. Other than that I think you did great !!!!!

Reply:
Ugh yeah I was having a lot of trouble with that yesterday, but think I managed to fix it just now? Thanks!!